db/schema.rb

```diff
@@ -10,12 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_01_28_095151) do
+ActiveRecord::Schema[7.0].define(version: 2023_01_28_112856) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "comments", force: :cascade do |t|
-    t.string "text"
+    t.text "text"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.bigint "post_id", null: false
```

Comment:
- [ ] Please add the reference for the author in both comments and posts tables as requested by the previous reviewer. 🙏